VolList.h

```diff
@@ -22,14 +22,16 @@ class VolList
 	// Load all identified vol files into Outpost 2's memory.
 	void LoadVolFiles();
 
+	static const unsigned MaxVolumeCount = 31;
+
 private:
 	std::vector<std::vector<char>> volPaths;
 	unsigned int volFileCount;
 	VolSearchEntry* volSearchEntryList;
 	
 	// Static buffer, to avoid dynamic memory allocation before heap is initialized
-	static const unsigned int VolSearchBufferSize = 32;
-	VolSearchEntry buffer[VolSearchBufferSize];
+	// buffer size must include an extra entry for an end of list marker 
+	VolSearchEntry buffer[MaxVolumeCount + 1];

```

Comment:
Comment capitalization is different here. Should we call the last entry a "null entry" or a "terminator entry"?

Reply:
Thanks for fixing the punctuation inconsistency. I think you are correct to say terminator as it explains better what is going on than saying null.